ace.d.ts

```diff
@@ -875,14 +875,56 @@ declare module "ace-code" {
             alignCursors: () => void;
             multiSelect?: any;
         }
+        /**
+         * Provider interface for code lens functionality
+         */
         interface CodeLenseProvider {
+            /**
+             * Compute code lenses for the given edit session
+             * @param session The edit session to provide code lenses for
+             * @param callback Callback function that receives errors and code lenses
+             */
             provideCodeLenses: (session: EditSession, callback: (err: any, payload: CodeLense[]) => void) => void;
         }
+
+        /**
+         * Represents a command associated with a code lens
+         */
+        interface CodeLenseCommand {
+            /**
+             * Command identifier that will be executed
+             */
+            id: string,
+            /**
+             * Display title for the code lens
+             */
+            title: string,
+            /**
+             * Arguments to pass to the command when executed
+             */
+            arguments: any[],
+        }
+
+        /**
+         * Represents a code lens - an actionable UI element displayed above a code line
+         */
         interface CodeLense {
-            start: Point;
-            command: any;
+            /**
+             * Starting position where the code lens should be displayed
+             */
+            start: Point,
+            /**
+             * Command to execute when the code lens is activated
+             */
+            command: CodeLenseCommand
```

Comment:
changing the type here from `any` to `CodeLenseCommand` could technically be seen as a breaking change, do you think there is a likely scenario where consumers would be using any other type than `CodeLenseCommand` here?